src/components/Charts/ClassCountDisplay.tsx

```diff
@@ -0,0 +1,21 @@
+import { FC, useEffect } from 'react';
+import { useAppDispatch, useAppSelector } from '../../redux/hooks';
+import { fetchClasses } from '../../redux/thunks/classesThunks.ts';
+
+const ClassCountDisplay: FC = () => {
+  const dispatch = useAppDispatch();
+  const { totalCount, loading, error } = useAppSelector(
+    (state) => state.classes
```

Comment:
Ja radši dělám selectory přímo v slice a pak je jen využívam. Je to vyhodné kdyby náhodou měníš strukturu slicu, tak pak nemusíš všude v komponentach upravovat